try:

 def additoin(x, y):
    x = 10
    y = 20
    print("Addition:", x + b)
    additoin(10, 20)


except NameError:
    print("Please check variables")
except :
    print(" please try again): ")
else:
    print("the operation is successful")
